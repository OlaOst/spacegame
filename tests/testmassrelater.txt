parent.mass = 2.0
parent.originalMass = 2.0
parent.isRelation = true

child.mass = 1.0
child.relationName = parent

otherchild.mass = 0.5
otherchild.relationName = parent

childchild.mass = 1.5
childchild.originalMass = 1.5
childchild.relationName = child