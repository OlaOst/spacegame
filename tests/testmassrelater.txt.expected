parent.mass = 3.5
parent.originalMass = 2
parent.isRelation = true

child.mass = 1
child.relationName = parent

otherchild.mass = 0.5
otherchild.relationName = parent

childchild.mass = 2.5
childchild.originalMass = 1.5
childchild.relationName = child